pkg/mcoproducer/testdata/mco-test.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: machineconfiguration.openshift.io/v1
```

Comment:
Can we rename this file to `machineconfig.yaml`?

Reply:
renaming it to machibeconfig-test.yaml